backend/src/test/scala/com/softwaremill/adopttapir/starter/StarterServiceITTest.scala

```diff
@@ -6,7 +6,6 @@ import cats.effect.unsafe.implicits.global
 import cats.instances.list._
```

Comment:
I think that splitting the file into smaller one could increase the readability. 

WDYT?